README.md

```diff
@@ -52,6 +52,13 @@ describe('analyze', () => {
         () => done()
       );
   });
+
+  // This test uses chai-as-promised for async assert.
+  // Ensure it is set up.
+  it('properly isolates the call to axe.finishRun', () => {
+    await driver.get('${addr}/isolated-finish.html')
+    assert.isRejected(new AxeBuilder(driver).analyze(), /PWNED/)
```

Comment:
For our non-js environments what is the appropriate library or code that duplicates this test?

Reply:
The gist is that running axe will cause the JS to throw. You should do the appropriate thing to check that that occurred.

Reply:
Whoops. Inverted the expectation. The real answer is that test will *not* throw if you've isolated properly. So you should just run axe and ensure no error as appropriate in the integration.